src/Arcus.Security.Core/Storage/CompositeSecretProvider.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
```

Comment:
Let's move this to `Arcus.Security.Core`

Reply:
Yes, that might be right!